### PR TITLE
VAL-64. Multysite settings

### PR DESCRIPTION
**Pull request by AndrewFall**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/107/

**Build Log**
*last 25 lines*

 ```
[...truncated 123 lines...]
changed: [localhost] => (item={'status': True, 'name': '$conf["cache_class_cache_form"]', 'value': 'DrupalDatabaseCache'})
changed: [localhost] => (item={'status': True, 'name': '$conf["stage_file_proxy_origin"]', 'value': u'https://favrskov.dk'})

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost]

TASK: [Adding domain configuration file to settings.php] ********************** 
changed: [localhost]

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
failed: [localhost] => {"failed": true}
msg: unable to connect, check login credentials (login_user, and login_password, which can be defined in ~/.my.cnf), check that mysql socket exists and mysql server is running

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=20   changed=14   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by AndrewFall** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/108/

**Build Log**
*last 25 lines*

 ```
[...truncated 142 lines...]
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
changed: [localhost]

TASK: [Importing from local backup] ******************************************* 
failed: [localhost] => {"failed": true}
msg: ERROR 1114 (HY000) at line 18052: The table 'watchdog' is full


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=23   changed=16   unreachable=0    failed=1   

Build step 'Execute shell' marked build as failure
```

**Comment by AndrewFall** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/109/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/110/

**Comment by AndrewFall** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/111/

File: drupal/subsite.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+- name: Inserting domain alias
+  shell: "mysql -u{{ mysql_user }} -p{{ mysql_pass }} {{ mysql_db }} -e 'insert into domain_alias (domain_id, pattern) values ({{ domain_id }}, \"{{ code }}{{ env_domain }}\");'"
```
Comment (temoor-gilmutdinov) [inline drupal/subsite.yml]: Check `drush sqlq`

File: provisioning/ansible/playbooks/www.yml [hunk 2]
```diff
@@ -28,6 +28,9 @@
       - {
         servername: "favrskov.loc",
         serveralias: "*.*",
+        serveralias1: "designguiden.favrskov.loc",
```
Comment (temoor-gilmutdinov) [inline provisioning/ansible/playbooks/www.yml]: Is that used properly in Apache configuration?

File: drupal/subsite.yml (hunk 1 above)
Comment (AndrewFall) [reply drupal/subsite.yml]: changed

File: provisioning/ansible/playbooks/www.yml (hunk 2 above)
Comment (AndrewFall) [reply provisioning/ansible/playbooks/www.yml]: yep

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/112/

**Build Log**
*last 25 lines*

 ```
[...truncated 234 lines...]
TASK: [Run general drush commands] ******************************************** 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"update apachesolr_environment set url=\'http://localhost:8983/solr/default\' where env_id=\'solr\'"'})

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"UPDATE users SET name=\'admin\' WHERE uid=1"'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=propeople'})

TASK: [Enable general modules] ************************************************ 
changed: [localhost]

TASK: [Enable environment specific modules] *********************************** 
changed: [localhost]

TASK: [Inserting domain alias] ************************************************ 
fatal: [localhost] => with_items expects a list or a set

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=45   changed=32   unreachable=1    failed=0   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/113/

**Build Log**
*last 25 lines*

 ```
[...truncated 234 lines...]
TASK: [Run general drush commands] ******************************************** 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"update apachesolr_environment set url=\'http://localhost:8983/solr/default\' where env_id=\'solr\'"'})

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"UPDATE users SET name=\'admin\' WHERE uid=1"'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=propeople'})

TASK: [Enable general modules] ************************************************ 
changed: [localhost]

TASK: [Enable environment specific modules] *********************************** 
changed: [localhost]

TASK: [Inserting domain alias] ************************************************ 
fatal: [localhost] => One or more undefined variables: 'unicode object' has no attribute 'domain_id'

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=45   changed=32   unreachable=1    failed=0   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/114/

**Build Log**
*last 25 lines*

 ```
[...truncated 234 lines...]
TASK: [Run general drush commands] ******************************************** 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"update apachesolr_environment set url=\'http://localhost:8983/solr/default\' where env_id=\'solr\'"'})

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"UPDATE users SET name=\'admin\' WHERE uid=1"'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=propeople'})

TASK: [Enable general modules] ************************************************ 
changed: [localhost]

TASK: [Enable environment specific modules] *********************************** 
changed: [localhost]

TASK: [Inserting domain alias] ************************************************ 
fatal: [localhost] => One or more undefined variables: 'unicode object' has no attribute 'domain_id'

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=45   changed=32   unreachable=1    failed=0   

Build step 'Execute shell' marked build as failure
```

**Comment by AndrewFall** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/115/

**Build Log**
*last 25 lines*

 ```
[...truncated 234 lines...]
TASK: [Run general drush commands] ******************************************** 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"update apachesolr_environment set url=\'http://localhost:8983/solr/default\' where env_id=\'solr\'"'})

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"UPDATE users SET name=\'admin\' WHERE uid=1"'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=propeople'})

TASK: [Enable general modules] ************************************************ 
changed: [localhost]

TASK: [Enable environment specific modules] *********************************** 
changed: [localhost]

TASK: [Inserting domain alias] ************************************************ 
fatal: [localhost] => One or more undefined variables: 'unicode object' has no attribute 'domain_id'

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=45   changed=32   unreachable=1    failed=0   

Build step 'Execute shell' marked build as failure
```

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/116/

**Build Log**
*last 25 lines*

 ```
[...truncated 234 lines...]
TASK: [Run general drush commands] ******************************************** 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"update apachesolr_environment set url=\'http://localhost:8983/solr/default\' where env_id=\'solr\'"'})

TASK: [Run environment specific drush commands] ******************************* 
changed: [localhost] => (item={'status': True, 'name': 'sqlq', 'arguments': u'"UPDATE users SET name=\'admin\' WHERE uid=1"'})
changed: [localhost] => (item={'status': True, 'name': 'user-password', 'arguments': u'admin --password=propeople'})

TASK: [Enable general modules] ************************************************ 
changed: [localhost]

TASK: [Enable environment specific modules] *********************************** 
changed: [localhost]

TASK: [Inserting domain alias] ************************************************ 
fatal: [localhost] => One or more undefined variables: 'unicode object' has no attribute 'domain_id'

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=45   changed=32   unreachable=1    failed=0   

Build step 'Execute shell' marked build as failure
```

**Comment by AndrewFall** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/117/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/118/

**Pull request merged**